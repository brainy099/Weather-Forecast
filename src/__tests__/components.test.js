import React from "react";
import renderer from "react-test-renderer";
import { fromJS } from "immutable";
import PlotComponent from "../components/Plot.component";

describe("components", () => {
  describe("<PlotComponent />", () => {
    global.Plotly = {
      newPlot: () => {}
    };

    global.document = {
      getElementById: () => {
        return {
          on: () => {}
        };
      }
    };

    it("renders correctly", () => {
      const tree = renderer
        .create(<PlotComponent xData={fromJS({})} yData={fromJS({})} />)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
