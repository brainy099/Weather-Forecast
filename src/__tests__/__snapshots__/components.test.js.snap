// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <PlotComponent /> renders correctly 1`] = `
<div
  style={
    Object {
      "display": "inline-block",
      "position": "relative",
    }
  }
/>
`;
